docker-php-source

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+set -e
+PHP_SRC_DIR=/usr/src/php
```

Comment:
Since this variable isn't intended to live beyond this script (and is thus a "local var" not an "env var"), convention ought to be `phpSrcDir` or even just `dir`. :+1: